httpx/models.py

```diff
@@ -667,14 +669,12 @@ def __init__(
         stream: ContentStream = None,
         content: bytes = None,
         history: typing.List["Response"] = None,
-        elapsed: datetime.timedelta = None,
     ):
         self.status_code = status_code
         self.http_version = http_version
         self.headers = Headers(headers)
 
         self.request = request
-        self.elapsed = datetime.timedelta(0) if elapsed is None else elapsed
         self.call_next: typing.Optional[typing.Callable] = None
 
         self.history = [] if history is None else list(history)
```

Comment:
We probably also want to add `self._elapsed = datetime.timedelta(0)` inside the `if stream is None` block.